test/extensions/bootstrap/wasm/wasm_speed_test.cc

```diff
@@ -59,7 +59,7 @@ static void BM_WasmSimpleCallSpeedTest(benchmark::State& state, std::string test
   auto vm_key = "";
   auto plugin_configuration = "";
   auto plugin = std::make_shared<Extensions::Common::Wasm::Plugin>(
-      name, root_id, vm_id, plugin_configuration, false,
+      name, root_id, vm_id, vm, plugin_configuration, false,
```

Comment:
Nit: might be worth changing `vm` to `runtime` to be consistent across the codebase.

Reply:
Done.